lua/graph.lua

```diff
@@ -996,6 +1038,10 @@ function filter_tags_generic(kv)
     if kv["motorroad"] == "yes" then
       kv["motorroad_tag"] = "true"
     end
+
+    if kv["golf_cart"] == "designated" then
+      kv["golf_cart_designated"] = "true"
```

Comment:
Worth noting: this felt a bit ad hoc when I wrote it, but I'm also not sure we want to introduce a generalized differentiation between designated and permissive across all costing modes.

In the case of golf carts, it actually is significant, since there are typically local ordinances to the effect of "use paths designated for golf cart use whenever possible, even though another road may be legally traversable for short distances"

Reply:
This has been removed.